cmd/gardener-controller-manager/app/app.go

```diff
@@ -123,8 +123,9 @@ func run(ctx context.Context, log logr.Logger, cfg *config.ControllerManagerConf
 
 	log.Info("Setting up manager")
 	mgr, err := manager.New(restConfig, manager.Options{
-		Logger:                  log,
-		Scheme:                  kubernetes.GardenScheme,
+		Logger: log,
+		Scheme: kubernetes.GardenScheme,
+
 		HealthProbeBindAddress:  fmt.Sprintf("%s:%d", cfg.Server.HealthProbes.BindAddress, cfg.Server.HealthProbes.Port),
 		MetricsBindAddress:      fmt.Sprintf("%s:%d", cfg.Server.Metrics.BindAddress, cfg.Server.Metrics.Port),
 		GracefulShutdownTimeout: pointer.Duration(5 * time.Second),
```

Comment:
```suggestion
		Logger: log,
		Scheme: kubernetes.GardenScheme,
		GracefulShutdownTimeout: pointer.Duration(5 * time.Second),

		HealthProbeBindAddress:  fmt.Sprintf("%s:%d", cfg.Server.HealthProbes.BindAddress, cfg.Server.HealthProbes.Port),
		MetricsBindAddress:      fmt.Sprintf("%s:%d", cfg.Server.Metrics.BindAddress, cfg.Server.Metrics.Port),
```
There is a tiny difference compare to the others 😅